eksupgrade/src/preflight_module.py

```diff
@@ -338,13 +331,13 @@ def addon_version(
                     "env": daemon_set_item.spec.template.spec.containers[0].env,
                 }
                 version = version_str.split(".")
-                logger.info("VPC CNI found version: %s - version_str: %s", version, version_str)
-                logger.info("Likely desired vpc-cni version: %s", cni_target_version)
+                echo_info(f"VPC CNI found version: {version} - version_str: {version_str}")
```

Comment:
- We should drop the `version` from the log as it is a list which can be confusing ( like in the discussion [here](https://github.com/aws-samples/eks-cluster-upgrade/discussions/85#discussioncomment-5362567)) 

`echo_info(f"VPC CNI found version: {version_str}")`

Reply:
Updated.